schemas/deviceToCloud/alert/alert.json

```diff
@@ -0,0 +1,60 @@
+{
```

Comment:
@savvyintegrations Question for you.

We don't have a `messageType` defined for this schema. I think Pete I were unsure why we needed it for this case and were discussing leaving it out. Do you see any issue with this? Could we leave it out?

Reply:
Same goes for Logs

Reply:
We don't need it on the device side.

Reply:
I don't see any issues off the top of my head. Seeing as most if not all other message types have this might cause issues with the code that processes these messages, but I didn't check. Worth verifying.

Reply:
I need to double check but I think there is no issues.

Reply:
I tested that we can handle  alert/log messages without messageType. There is not issue with it. So we will not add it here.